distributed-commandbus-jgroups/src/main/java/org/axonframework/jgroups/commandhandling/JGroupsConnector.java

```diff
@@ -238,7 +233,7 @@ public synchronized void viewAccepted(final View view) {
 
     @Override
     public void suspect(Address suspected_mbr) {
```

Comment:
Why not a camel case?

Reply:
Apparently this is how the overridden function is specified..although it annoys me like hell too.
Think I'll adjust it any how, thanks for sharing this concern with me.